### DISCUSSION TITLE
The New Updated Search Box Bug

### DISCUSSION THREAD

**Discussion by TimMarvelHero** [valinet/ExplorerPatcher#2849] [General]
When you open and search something, the search box disappears.

Without searching anything:
<img width="304" alt="image" src="https://github.com/valinet/ExplorerPatcher/assets/152800265/0efa7619-343a-4672-b522-e8bcc41fce85">

When you press to search on it:
<img width="297" alt="image" src="https://github.com/valinet/ExplorerPatcher/assets/152800265/67bad9c0-1e53-4b89-967d-89585c2a8253">

**Comment by pyrates999** [comment]
Please post windows version and build.  You can find this by running winver.
Please post explorer patcher version.

Please include what settings you changed in explorer patcher properties application.  These should be screen shots.

Please include any other software you have installed that modifies the UI of windows.

**Comment by milestprower92** [comment]
that happens because windows 11 uses a different search program and we don't have plans to recreate it

**Comment by Amrsatrio** [comment]
As said in the release notes, please do not make new Issues (in extension Discussions too) regarding this. This is very hard to fix properly, especially because of what milesprower2293 said, but because people BEGGED me for the search box I had to at least make it show.

I think I have to disable this again because people expect proper functionality...

**Reply by haydengriz** [reply]
just put a warning on it or something like next to it in the settings

**Reply by TimMarvelHero** [reply]
It's better than nothing.

**Reply by TimMarvelHero** [reply]
Although one thing that bugs me out is that it already disappears when I just press on Windows even though I am not searching anything yet. If it can't be fixed then it's fine, you tried your best.

**Reply by milestprower92** [reply]
it disappears on the start menu because it uses the quick search function (it opens an invisible search menu that becomes visible when you start typing)

**Reply by pyrates999** [reply]
I think it should be hidden until it can be properly implemented.

**Reply by TimMarvelHero** [reply]
They've said it's a pain to implement it.

**Reply by pyrates999** [reply]
then it will never be shown.

**Reply by TimMarvelHero** [reply]
But making the big search box at least show is already good.

**Reply by pyrates999** [reply]
not if it has a glaring bug in it

**Reply by TimMarvelHero** [reply]
I assume that was on purpose

**Comment by haydengriz** [comment]
I already posted this issue
